include/deal.II/integrators/advection.h

```diff
@@ -257,6 +349,136 @@ namespace LocalIntegrators
         }
     }
 
+
+
+    /**
+     * Scalar case: Residual for upwind flux at the boundary for weak advection operator. This is the
+     * value of the trial function at the outflow boundary and zero else:
+     * @f[
+     * a_{ij} = \int_{\partial\Omega}
+     * [\mathbf w\cdot\mathbf n]_-
+     * (u_i-g) v_j \, ds
+     * @f]
```

Comment:
I guess that's the same case as below where you actually want the inflow boundary.